lib/index.js

```diff
@@ -1,21 +1,50 @@
 var AWS = require('aws-sdk');
 AWS.config.region = process.env.AWS_REGION;
+
+
+
```

Comment:
please add `/** JSDoc Comment documenting this new function ...**/` 
(_thanks_)
example: [/lib/extract.js#L5-L12](https://github.com/dwyl/hapi-auth-jwt2/blob/9abe5b3bc3533331f44f41040f87ca3c4abeb642/lib/extract.js#L5-L12)